docs/modules/localstack.md

```diff
@@ -7,7 +7,7 @@ Testcontainers module for the Atlassian's [LocalStack](https://github.com/locals
 Running LocalStack as a stand-in for AWS S3 during a test:
 
 ```java
-DockerImageName localstackImage = DockerImageName.parse("localstack/localstack:0.11.3");
+DockerImageName localstackImage = DockerImageName.parse("localstack/localstack:0.13.2");
```

Comment:
In general, we migrated most docs to use code examples from the tests themselves, via code include. This PR can be a great motivation to do this for the LocalStack module as well.